bin/node-template/node/src/service.rs

```diff
@@ -216,44 +223,11 @@ where
 		config.prometheus_registry(),
 		CanAuthorWithNativeVersion::new(client.executor().clone()),
 	)?;
-	let justification_stream = grandpa_link.justification_stream();
-	let shared_authority_set = grandpa_link.shared_authority_set().clone();
 	let shared_voter_state = GrandpaSharedVoterState::empty();
 	let finality_proof_provider =
 		GrandpaFinalityProofProvider::new_for_service(backend.clone(), client.clone());
 	let import_setup = (babe_import.clone(), grandpa_link, babe_link.clone());
 	let rpc_setup = (shared_voter_state.clone(), finality_proof_provider.clone());
-	let babe_config = babe_link.config().clone();
-	let shared_epoch_changes = babe_link.epoch_changes().clone();
-	let rpc_extensions_builder = {
```

Comment:
@AurevoirXavier pls help review this part, is it ok to remove it?

Reply:
Yes, these code pieces needed more attention. The `rpc_extensions_builder` need a parameter `network`.